smart-cache-enterprise/Dockerfile

```diff
@@ -1,9 +1,9 @@
-ARG BASE_IMAGE=registry.access.redhat.com/ubi8/openjdk-11
-ARG BASE_TAG=1.16-3
+ARG BASE_IMAGE=registry.access.redhat.com/ubi8-minimal
+ARG BASE_TAG=latest
 FROM ${BASE_IMAGE}:${BASE_TAG}
 # The argument below is no longer used. We are moving to ubi8/openjdk-11 image
 # instead of post-install of jdk due to tzdb corruption in upstream repo.
-#ARG JAVA_VERSION=java-11-openjdk
+ARG JAVA_VERSION=java-11-openjdk-headless
 ARG GS_PRODUCT=smart-cache
 ARG GS_VERSION=16.4.0
 ARG GS_BUILD_NAME=16.4.0-PIC-3160-tue-1139
```

Comment:
how about making this an argument, docker build ... --build-arg GS_BUILD_NAME=16.4.0-m9
since we keep updating the GS_BUILD_NAME env in the Dockerfile, and it creates a new intermediate layer each time.

Reply:
ok let's do these improvements incrementally